Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationOverview.cshtml

```diff
@@ -133,44 +135,44 @@
                 @if (organisationDetails.IsTenderer())
                 {
                     <h3 class="govuk-heading-s govuk-!-margin-bottom-3">
-                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/supplier-information">Complete supplier information</a>
+                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/supplier-information">@StaticTextResource.Organisation_CompleteSupplierInformation</a>
                     </h3>
-                    <p class="govuk-body">Complete and share your supplier information to bid for procurements.</p>
+                    <p class="govuk-body">@StaticTextResource.Organisation_CompleteAndShareSupplierInformation</p>
                     <div class="govuk-section-break govuk-section-break--visible govuk-section-break--m"></div>
                 }
 
                 @if (organisationDetails.IsBuyer())
                 {
                     <h3 class="govuk-heading-s govuk-!-margin-bottom-3">
-                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/manage-api-keys">@(isViewer ? "View" : "Manage") API keys@(organisationDetails.IsTenderer() ? " (for Buyers)" : "")</a>
+                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/manage-api-keys">@StaticTextResource.Organisation_ManageApiKeys@(isViewer ? " @StaticTextResource.Global_View" : " @StaticTextResource.Global_Manage")</a>
```

Comment:
This should be two distinct keys, switched based on isViewer. I'm not quite sure what's going on here anyway, with one bit of text and the beginning and then view / manage stuck on the end?

```
@StaticTextResource.Organisation_ManageApiKeys@(isViewer ? " @StaticTextResource.Global_View" : " @StaticTextResource.Global_Manage")
```

Again, I don't think this is a reasonable assumption about the Welsh language - we can't construct sentences from smaller pieces because Welsh won't always work the same as English does.

Reply:
Done